network/server.go

```diff
@@ -149,6 +149,10 @@ func NewServer(logger hclog.Logger, config *Config) (*Server, error) {
 			if err != nil {
 				return nil, fmt.Errorf("failed to parse bootnode %s: %v", raw, err)
 			}
+			if node.ID == srv.host.ID() {
+				srv.logger.Info("Omitting bootnode with same ID as host", node.ID)
```

Comment:
This can probably be a `Warning` instead of an `Info` log

Reply:
I felt `Info` makes more sense, since this bootnode entry would be a valid one for other nodes in the cluster.

Reply:
Agree with @munna0908 this is a low-severity message.